frontends/web/src/routes/device/bitbox02/passphrase.tsx

```diff
@@ -251,11 +351,17 @@ class Passphrase extends Component<Props, State> {
                                     <SimpleMarkup key="tip-2" tagName="li" markup={t('passphrase.successEnabled.tipsList.1')} />
                                 </ul>
                             )}
-                        </WaitDialog>
-                    )}
-                    </div>
-                </div>
-            </div>
+                            <SimpleMarkup tagName="p" markup={t(
+                                passphraseEnabled
+                                    ? 'passphrase.successDisabled.messageEnd'
+                                    : 'passphrase.successEnabled.messageEnd')} />
+                        </ViewContent>
+                    </View>
+                )}
+                {/* <Guide>
+                    <Entry key="whatisapassphrase" entry={t('guide.passphrase.whatisapassphrase')} />
+                </Guide> */}
```

Comment:
remove this since there's no guide anymore?

Reply:
I really'd like the to keep it here until we have another example that uses `<View>` and a guide component.